packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx

```diff
@@ -131,22 +131,25 @@ const DateFilterInputs: FC<FilterInputsProps<DateFilterRule>> = (props) => {
                     className={disabled ? 'disabled-filter' : ''}
                     disabled={disabled}
                     fill
-                    showTimezoneSelect={true}
                     value={
                         filterRule.values?.[0]
-                            ? new Date(filterRule.values?.[0]).toString()
+                            ? formatDate(
+                                  filterRule.values?.[0],
+                                  undefined,
```

Comment:
A problem when filtering from results
![filter timestamp](https://user-images.githubusercontent.com/9117144/198015987-6e27b839-b688-43e6-8ca7-88b3ce61c6fa.gif)

Perhaps we need to pass the time interval when we call formatDate.

Reply:
[Screencast from 26-10-22 14:20:12.webm](https://user-images.githubusercontent.com/1983672/198025421-ae3a4739-6007-478a-8852-b2d55d5efd44.webm)